lib/commands.sh

```
@@ -163,21 +270,50 @@ cmd_policy() {
 }
 
 usage() {
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this function.

(SC1009)